src/language-server/language-server.ts

```diff
@@ -87,14 +118,12 @@ export async function startLanguageServer() {
   const documentChangeSubscription = documentChangeObservable
     .pipe(
       debounceTime(200),
-      map((_) => buildWorkspaceDiagnosticsMap(connection, workspaceFolder, documents)),
+      map(async (_) => {
+        return await buildWorkspaceDiagnosticsMap(connection, workspaceFolder, documents, puyaService!)
+      }),
       concatMap(async (v) => sendDiagnostics(connection, await v)),
     )
-    .subscribe(async () => {
-      // All logic for handling the document change event is done inside the pipe
-      // This is to make sure that the diagnostics are sent in the right order
-      // The empty subscribe function here is to make the observable run
-    })
+    .subscribe()
```

Comment:
Happy to drop the handler function, however can we keep the comment about not having a handler.